Mathlib/Data/ZMod/Defs.lean

```diff
@@ -43,6 +43,26 @@ to register the ring structure on `ZMod n` as type class instance.
 
 open Nat.ModEq Int
 
+open scoped Fin.IntCast Fin.NatCast
+
+/-- Variant of `Fin.intCast_def` for when we are using the `Fin.CommRing` instance. -/
+theorem intCast_def' {n : Nat} [NeZero n] (x : Int) :
```

Comment:
Since this isn't the def, should be call it `intCast_eq_if`? My impression is that primed names are discouraged now.

Reply:
Oh I see, it's just moved. Fine as is then.

Reply:
Agreed that this is a non-ideal name, but fixing that is a job for a separate PR